src/app/connections/state/connections.state.ts

```diff
@@ -148,7 +148,8 @@ export class ConnectionsState extends RecordsState {
       filter: action.filter,
       disableSearchAndSort:
         action.filter !== ConnectionsFilterType.ALL &&
-        action.filter !== ConnectionsFilterType.HIDDEN
+        action.filter !== ConnectionsFilterType.HIDDEN &&
+        action.filter !== ConnectionsFilterType.EXCEPTIONS_QUEUE
```

Comment:
Nothing wrong with this but if we add more to this list it's probably easier/cleaner to filter types to disable rather than those '!=='